packages/app-bridge/src/AppBridgePlatformApp.ts

```diff
@@ -94,35 +98,45 @@ export class AppBridgePlatformApp implements IAppBridgePlatformApp {
     api<ApiMethodName extends keyof PlatformAppApiMethod>(
         apiHandler: ApiHandlerParameter<ApiMethodName, PlatformAppApiMethod>,
     ): ApiReturn<ApiMethodName, PlatformAppApiMethod> {
-        return this.messageBus.post({
-            method: 'api',
+        return this.apiMessageBus.post({
             parameter: apiHandler,
         }) as ApiReturn<ApiMethodName, PlatformAppApiMethod>;
     }
 
+    private initializationGuard() {
```

Comment:
As it's a method name, let's use imperative mode, e.g. `checkInitialization()`

Reply:
Thats a very good point. 

The idea with this function is to Guard against programm execution. When not all the statements are fulfilled it should abort (silently or with an error in this case). Here we use [Guard as a Pattern](https://en.wikipedia.org/wiki/Guard_(computer_science)). let me think a moment if we can do both, be explicit about it beeing a guard and having a verb involved. If you have any idea im a tacker :D

Reply:
Thanks for elaborating! 

'guard' itself is also a verb, perhaps something starting with `guardFor..`?